### DISCUSSION TITLE
Note/RequestDoc: Obscenity and Bad Language detection

### DISCUSSION THREAD

**Discussion by BradKML** [textlint/textlint#753] [Ideas]
Here is a list of repositories for profanity databases that can be used for filtering and substitution.

- https://github.com/LDNOOBW/List-of-Dirty-Naughty-Obscene-and-Otherwise-Bad-Words
- https://github.com/RobertJGabriel/Google-profanity-words
- https://github.com/MauriceButler/badwords
- https://github.com/words/cuss#support
- https://github.com/thisandagain/washyourmouthoutwithsoap
- https://github.com/chucknorris-io/swear-words
- https://github.com/retextjs/retext-profanities#related

**Comment by azu** [comment]
Nice! It's good sources.

**Reply by BradKML** [reply]
Thank you! I do think that community collaboration is healthy for this type of thing.
